AsyncAwait.Task1.CancellationTokens/Program.cs

```diff
@@ -17,45 +19,52 @@ internal class Program
     /// The Main method should not be changed at all.
     /// </summary>
     /// <param name="args"></param>
-    private static void Main(string[] args)
+    private static async Task Main(string[] args)
     {
         Console.WriteLine("Mentoring program L2. Async/await.V1. Task 1");
         Console.WriteLine("Calculating the sum of integers from 0 to N.");
         Console.WriteLine("Use 'q' key to exit...");
         Console.WriteLine();
 
-        Console.WriteLine("Enter N: ");
+        AskN();
 
         var input = Console.ReadLine();
         while (input.Trim().ToUpper() != "Q")
         {
+            CancellationTokenSource cancellationTokenSource = new();
             if (int.TryParse(input, out var n))
             {
-                CalculateSum(n);
+                CalculateSum(n, cancellationTokenSource.Token);
```

Comment:
Please add task await is some appropriate place. It's a bad practice to leave tasks unawaited

Reply:
Fixed